pkg/operators/localmanager/localmanager.go

```diff
@@ -486,8 +462,8 @@ func (l *localManagerTrace) PostGadgetRun() error {
 			}
 		}
 	}
-	if l.containersSubscriptionKey != "" {
-		l.manager.igManager.Unsubscribe(l.containersSubscriptionKey)
+	if l.containersPublisher != nil {
+		l.containersPublisher.Unsubscribe()
```

Comment:
nit: I rather move this to Stop() as this logic is only relevant for image-based gadgets. (Yes, I know having the two different kind of hooks on the same file is a mess)